cmd/githubgen/main.go

```diff
@@ -75,13 +78,28 @@ internal/common
 
 const unmaintainedStatus = "unmaintained"
 
+//go:embed members.txt
+var membersData string
+
+//go:embed allowlist.txt
+var allowlistData string
+var members []string
+var allowlist []string
```

Comment:
Why not read these at runtime? Recompiling the app every time we change these files doesn't seem ideal and makes us have to leave the app on this repository instead of eventually moving it to opentelemetry-go-build-tools

Reply:
It's not in my scope to move this code to opentelemetry-go-build-tools. This code was definitely not designed to be portable, and maybe someone else can work on this. The code runs with a `go run` command right now in the Makefile, effectively embedding the files at the time.

Reply:
I am not saying to move this to opentelemetry-go-build-tools now, I am just saying that we should not make it harder to eventually move it by embedding the files. Even if we don't ever move it, embedding the files is probably confusing for people editing said files

Reply:
OK it's now readable and we have an argument to make this configurable